vercel.json

```diff
@@ -423,6 +423,146 @@
+      "destination": "https://www.ory.sh/docs/guides/social-signin/overview",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/credentials/totp",
+      "destination": "https://www.ory.sh/docs/guides/mfa/totp",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/credentials/webauthn",
+      "destination": "https://www.ory.sh/docs/guides/mfa/webauthn-fido-yubikey",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/federation",
+      "destination": "https://www.ory.sh/docs/kratos/",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/management-ui",
+      "destination": "https://www.ory.sh/docs/welcome",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/rest-api",
+      "destination": "https://www.ory.sh/docs/ecosystem/api-design",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/securing-applications",
+      "destination": "https://www.ory.sh/docs/kratos/",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/concepts/terminology",
+      "destination": "https://www.ory.sh/docs/concepts/terminology",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/further-reading/comparison",
+      "destination": "https://www.ory.sh/docs/kratos/",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/further-reading/kratos-video-tutorials",
+      "destination": "https://www.ory.sh/docs/kratos/quickstart",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/guides/retrieve-social-sign-in-access-refresh-id-token",
+      "destination": "https://www.ory.sh/docs/guides/social-signin/get-tokens",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/guides/setting-up-aes-cipher-parameters",
+      "destination": "https://www.ory.sh/docs/kratos/guides/select-cipher-algorithm",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/guides/setting-up-noop-cipher-parameters",
+      "destination": "https://www.ory.sh/docs/kratos/guides/select-cipher-algorithm",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/guides/setting-up-xchacha-cipher-parameters",
+      "destination": "https://www.ory.sh/docs/kratos/guides/select-cipher-algorithm",
+      "permanent": false
+    },
+    {
+      "source": "/docs/kratos/self-service/flows/2fa-mfa-multi-factor-authentication",
+      "destination": "https://www.ory.sh/docs/guides/mfa/overview",
+      "permanent": false
+    },
+    {
+      "source": "/docs/start-building/deploy-auth-production",
+      "destination": "https://www.ory.sh/docs/guides/custom-domains",
+      "permanent": false
+    },
+    {
+      "source": "/docs/start-building/other-languages",
+      "destination": "https://www.ory.sh/docs/guides/protect-page-login/go",
```

Comment:
this could be changed to the example page when we merge that.

Reply:
Yes, that's a good idea!